### ISSUE TITLE
Configuring joystick causes segfault

### ISSUE THREAD

**Issue by etam** [jukibom/FlyDangerous#135]
I have a Saitek X45 joystick.
OS: Linux, openSUSE Tumbleweed.
Reproducible: always

Steps to reproduce:
- launch the game
- go to settings
- choose advanced input
- bindings
- ship rotation
- pitch full axis
- click "(unbound secondary)"
- game waits for an input
- move the joystick
- game crashes immediately

**Comment by jukibom** [comment]
@etam that's a new one! Is there anything in the log file at `~/.config/unity3d/StarGoat/FlyDangerous/Player.log`?

**Comment by etam** [comment]
Yes: [Player.log](https://github.com/jukibom/FlyDangerous/files/8720911/Player.log)

**Comment by jukibom** [comment]
Interesting, looks like a unity issue - I'll report that, thanks!

**Comment by jukibom** [comment]
Two more bug reports of the same - this isn't likely a linux issue anymore, and should be tested with fresh preferences json.

**Comment by jukibom** [comment]
Yep confirmed this is a float => string culture encoding issue on the deadzone processor
